docs-web/src/main/webapp/src/partial/docs/document.default.html

```diff
@@ -76,6 +76,7 @@ <h3 class="well-3d-header text-muted">
         <thead>
         <tr>
           <th class="col-xs-6">{{ 'document.title' | translate }}</th>
+          <th class="col-xs-6">{{ 'document.status' | translate }}</th>
```

Comment:
change `document.status` to `document.current_step_status`, add `document.current_step_priority`

Reply:
I have added the changes to the HTML file